File: insight/src/test/testData/java/DeadCodeDuration.java
public class DeadCodeDuration {
    public void code1() {
        if (false) {
            Thread.sleep(200);
        }
        Thread.sleep(200);
    }

    public void code2() {
        if (false) {
            Thread.sleep(200);
        } else {
            Thread.sleep(200);
            Thread.sleep(200);
        }
        Thread.sleep(200);
    }
}

File: insight/src/test/testData/kotlin/DeadCodeDuration.kt
public class DeadCodeDuration {
    fun code1() {
        if (false) {
            Thread.sleep(200)
        }
        Thread.sleep(200)
    }

    fun code2() {
        if (false) {
            Thread.sleep(200)
        } else {
            Thread.sleep(200)
            Thread.sleep(200)
        }
        Thread.sleep(200)
    }
}